app/assets/stylesheets/views/_homepage.scss

```
@@ -186,6 +186,14 @@
   @include govuk-media-query($from: "desktop") {
     @include columns($items: 16, $columns: 3, $selector: "li");
   }
+
+  .homepage-section__government_activity & {
```

It'd be good to follow the BEM convention of using hyphens in element name rather than underscores:

```suggestion
  .homepage-section__government-activity & {
```